scripts/ci

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+cd $(dirname $0)
+
+./build
+./test
+./validate
```

Comment:
We should probably run `validate` first, then run `test`, THEN run `build`. No sense building and stuff if it fails validation or tests.

Reply:
Rancher CI is running `build` first and `test` after. I agree on run `validate` first.

Reply:
The `test` process requires the binary from the `build` process. That's why we need to build first. On the other hand, it would be meaningless for `test` or `validate` if the `build` is failed.